### PR TITLE
tests: start integration of Jest tests

### PR DESCRIPTION
**Pull request by dackers86**

### CODE REVIEW

File: .github/workflows/tests_jest.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        id: yarn-cache
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/package.json') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+      - name: Yarn Install
+        run: yarn --no-audit --prefer-offline
+      - name: Jest
+        run: cd tests && yarn run tests:jest-coverage
```
Comment (Salakar) [inline .github/workflows/tests_jest.yml:33]:
```suggestion
        run: yarn run tests:jest-coverage
```

No longer in test directory

Comment (dackers86) [reply .github/workflows/tests_jest.yml]: Thanks. Updated

File: tests/.babelrc [hunk 2]
```diff
@@ -10,5 +10,10 @@
         "useInlineSourceMaps": false
       }
     ]
-  ]
+  ],
+  "env": {
+    "test": {
+      "presets": ["module:metro-react-native-babel-preset"]
+    }
+  }
```
Comment (Salakar) [inline tests/.babelrc:18]:
```suggestion
  ]
```

File: packages/functions/__tests__/functions.test.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import functions from '../lib';
+
+describe('functions', () => {
```
Comment (Salakar) [inline packages/functions/__tests__/functions.test.js:3]:
```suggestion
describe('Cloud Functions', () => {
```

**Pull request merged**